pandas/core/frame.py

```diff
@@ -6741,16 +6742,38 @@ def _gotitem(
         **_shared_doc_kwargs,
     )
     @Appender(_shared_docs["aggregate"])
-    def aggregate(self, func, axis=0, *args, **kwargs):
+    def aggregate(self, func=None, axis=0, *args, **kwargs):
         axis = self._get_axis_number(axis)
 
+        relabeling, func, columns, order = reconstruct_func(func, *args, **kwargs)
```

Comment:
is this function also used in groupby? did you already make the appropriate changes there?

Reply:
yeah, this is exactly the same as part of agg in `groupby/generic.py`, and later should be used for `resample/rolling` agg. Shall I replace the corresponding part in `groupby/generic.py` with this function in this PR? i was supposed to do that in a follow-up to make this PR smaller, but maybe it's better to do it here, will do then!